sigar_linux_common.go

```diff
@@ -89,6 +89,30 @@ func (self *Swap) Get() error {
 	return nil
 }
 
+func (self *HugeTLBPages) Get() error {
+	table, err := parseMeminfo()
+	if err != nil {
+		return err
+	}
+
+	self.Total, _ = table["HugePages_Total"]
+	self.Free, _ = table["HugePages_Free"]
+	self.Reserved, _ = table["HugePages_Rsvd"]
+	self.Surplus, _ = table["HugePages_Surp"]
+	self.DefaultSize, _ = table["Hugepagesize"]
+
+	if totalSize, found := table["Hugetlb"]; found {
```

Comment:
This value is [documented](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt), but I haven't seen it in any running Linux I could check now.